.gitignore

```diff
@@ -3,6 +3,7 @@ dist
 
 # Yarn/NPM
```

Comment:
We should probably remove Yarn from the comment too.

Reply:
I've made this change.